ci/prs.yml

```diff
@@ -47,14 +47,6 @@ jobs:
       env:
         PR: $(System.PullRequest.PullRequestNumber)
 
```

Comment:
Ledger api compatibility test will live in canton instead (https://github.com/DACH-NY/canton/issues/16637 )